internal/protocoltest/awsrestjson/api_client.go

```diff
@@ -117,6 +117,7 @@ func (c *Client) invokeOperation(ctx context.Context, opID string, params interf
 	ctx = middleware.ClearStackValues(ctx)
 	stack := middleware.NewStack(opID, smithyhttp.NewStackRequest)
 	options := c.options.Copy()
+
 	for _, fn := range optFns {
```

Comment:
Really minor but are we able to "fix" this new line begin added when not real change is needed?